cmd/import-config.go

```diff
@@ -138,6 +139,14 @@ func sshConfig(conf *config.Config) error {
 		return nil // nothing to do
 	}
 
+	sshDir := path.Dir(conf.ClientConfig.SSHPrivateKey)
+	sshConfPath := path.Join(sshDir, "config")
+	err := backupFile(sshConfPath, fmt.Sprintf("%s.bak", sshConfPath))
```

Comment:
this only works if there isn't already a .bak. To be really safe we probably need bak1, bak2, etc.

Reply:
put a timestamp instead